pilot/pkg/features/pilot.go

```diff
@@ -162,6 +162,13 @@ var (
 			"could result in an infinite loop of traffic. This option is only provided for backward compatibility purposes "+
 			"and will be removed in the near future.",
 	)
+
+	ScopePushes = env.RegisterBoolVar(
+		"PILOT_SCOPE_PUSHES",
+		true,
+		"If enabled, pilot will attempt to limit unnecessary pushes by determining what proxies "+
+			"a config or endpoint update will impact.",
```

Comment:
i dont think you need this because what you currently have is incremental eds

Reply:
Its not the same as incremental EDS I think. If a new update comes in, it will just push EDS, but it will push to all proxies right now. With this change it will only push to proxies that import with Sidecar

Reply:
> If a new update comes in, it will just push EDS, but it will push to all proxies right now. With this change it will only push to proxies that import with Sidecar

typo? I dont understand..

Reply:
are you saying if a new endpoint comes up, it will do a full push to all proxies, where most of the proxies wont even be receiving that endpoint?

Reply:
Currently yes, that is what this PR changes